FRBDK/Glue/Glue/CodeGeneration/ElementComponentCodeGenerator.cs

```diff
@@ -86,14 +86,14 @@ public virtual ICodeBlock GenerateDestroy(ICodeBlock codeBlock, IElement element
 
         #endregion
 
-        public virtual void GenerateActivityEditMode(ICodeBlock codeBlock, GlueElement element) {  }
+        public virtual void GenerateActivityEditMode(ICodeBlock codeBlock, IElement element) {  }
```

Comment:
This should remain GlueElement not IElement - we are slowly moving away from IElement.